sip-servlets-test-suite/applications/shootist-sip-servlet/src/main/java/org/mobicents/servlet/sip/testsuite/ShootistSipServlet.java

```diff
@@ -439,6 +439,12 @@ public void servletInitialized(SipServletContextEvent ce) {
 				addr.setParameter("headerparam1", "headervalue1");
 				addr.setParameter("param5", "ffff");
 				addr.getURI().setParameter("uriparam", "urivalue");
+				if (ce.getServletContext().getInitParameter("testContactHeader") != null){
+					addr.setParameter("param0", "value0");

```

Comment:
Hi @jaimecasero,
I have updated the code based on your comments. Please take a look again.

I also have pulled a request for jain-sip to support #110, please help me to review it as well.
Link: https://github.com/RestComm/jain-sip/pull/106